### PR TITLE
OSI Layer Model

### PR DESCRIPTION
**Pull request by Aju100**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+- A layer should only be created where definite levels of abstraction are needed.
+- The function of each layer should be selected as per the internationally standardized protocols.
+- The number of layers should be large so that separate functions should not be put in the same layer. At the same time, it should be small enough so that architecture doesn't become very complicated.
+- In the OSI model, each layer relies on the next lower layer to perform primitive functions. Every level should able to provide services to the next higher layer.
+- Changes made in one layer should not need changes in other lavers.
+ 
+As mention, the model uses layers to help give a visual description of what is going on with a particular networking system. This can help network managers narrow. 
+
+### Functions of the OSI Layers
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md:49]: Include the image for OSI model here.

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+- Physical Layer
+- Data-Link Layer
+- Network Layer
+- Transport Layer
+- Session Layer
+- Presentation Layer
+- Application Layer
+
+#### Physical layer
+- The main functionality of the physical layer is to transmit the individual bits from one node to another node.
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md]: If you mention it is the lowest layer, the reader might get confused looking at the order given above. Ensure you maintain consistency.

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- It is the lowest layer of the OSI model, establishes, maintains, and deactivates the physical connection.
+- It specifies the mechanical, electrical, and procedural network interface specifications.
+
+##### Functions of a Physical layer:
+- Line Configuration: It defines the way how two or more devices can be connected physically.
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md:65]: _Line Configuration_

Comment (lalith1403) [reply articles/lets-learn-about-osi-layer-model/index.md:65]: Do the same for all sub-titles mentioned in the article.

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+#### Network Layer
+- It is a third layer that manages device addressing, tracks the location of devices on the network.
+- It manages the delivery of individual data packets from source to destination through appropriate addressing and routing.
+- It determines the best path to move data from source to destination based on the network conditions, the priority of service, and other factors.
+- The Data link layer is responsible for routing and forwarding the packets.
+- Routers are the layer 3 devices, they are specified in this layer and used to provide the routing services within an internetwork.
+- The protocols used to route the network traffic are known as Network layer protocols. Examples of protocols are IP and Ipv6.
+
+##### Functions of Network Layer:
+- Internetworking: An internetworking is the main responsibility of the network layer. It provides a logical connection between different devices.
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md:91]: Inter-networking.

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+- The application layer interacts with an application program that is closest to the end-user. 
+- It allows users to interact with other software applications. 
+- The application layer interacts with software applications to implement a communicating component. 
+
+##### The function of the Application Layers are:
+- Application-layer helps you to identify communication partners, determining resource availability, and synchronizing communication.
+- It allows users to log on to a remote host and provides various e-mail services.
+- This application offers distributed database sources and access to global information about various objects and service
+ 
+Quick Way to memorize the OSI Model Layer concept - A Penguin Said That Nobody Drinks Pepsi
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md:130]: Italicize the mnemonic used.

File: articles/lets-learn-about-osi-layer-model/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+
```
Comment (lalith1403) [inline articles/lets-learn-about-osi-layer-model/index.md:132]: Add a Conclusion section, and conclude the article gracefully.

**Review by lalith1403 — CHANGES_REQUESTED** [review]

**Comment by Aju100** [comment]
@lalith1403  thank your for feedbacks i will add it as it's my first PR

File: articles/lets-learn-about-osi-layer-model/index.md (hunk 6 above)
Comment (Aju100) [reply articles/lets-learn-about-osi-layer-model/index.md]: @lalith1403 I have added it

File: articles/lets-learn-about-osi-layer-model/index.md (hunk 1 above)
Comment (Aju100) [reply articles/lets-learn-about-osi-layer-model/index.md:49]: @lalith1403  can you please review I have included the image

File: articles/lets-learn-about-osi-layer-model/index.md (hunk 3 above)
Comment (Aju100) [reply articles/lets-learn-about-osi-layer-model/index.md]: Yeah I hope I resolve it

**Comment by hectorkambow** [comment]
@Aju100  👋  Good evening and Welcome to the EngEd program. 
Happy to have you join us. 

Was getting ready to give your article a final review - but upon running it through our third party plagiarism checker it raised many red flags. (see PDF.)

We strive to make sure our content is wholly 
[Uploading OSI Layer Model #1073.pdf…]()
unique and original. 

Not sure if this was an older article published somewhere else.

We

**Pull request closed without merging**